### DISCUSSION TITLE
Renewing github student pack

### DISCUSSION THREAD

**Discussion by kennysofworld** [community/community#65380] [Other Feature Feedback, Questions, & Ideas] [answered]
GitHub Student Pack

Can the GitHub Student Pack be renewed? Will renewing grant access to the free services once more?

**Comment by pedromelobitencourt** [comment, marked as answer]
After the conclusion of your GitHub Global Campus access period, it's possible to submit a new application if you meet the eligibility criteria. However, it's important to note that certain partner benefits within the GitHub Student Developer Pack might not be available for renewal. The majority of the time-based offers from our partners initiate upon your setup activation.
